localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -280,6 +280,22 @@ void NDTScanMatcher::timer_diagnostic()
       diag_status_msg.level = diagnostic_msgs::msg::DiagnosticStatus::ERROR;
       diag_status_msg.message += "skipping_publish_num exceed limit. ";
     }
+    if(state_ptr_->count("nearest_voxel_transformation_likelihood") &&
+      std::stoi((*state_ptr_)["nearest_voxel_transformation_likelihood"]) < 2.3) {
+      diag_status_msg.level = diagnostic_msgs::msg::DiagnosticStatus::WARN;
+      diag_status_msg.message += "NDT score is unreliably low. ";
+    }
+    if(state_ptr_->count("exe_time") &&
+      std::stof((*state_ptr_)["exe_time"]) > 13 &&
+      std::stof((*state_ptr_)["exe_time"]) < 24) {
```

Comment:
Could you please consider implementing 24 as a const. 
As Koji said, it is better to make these ROS 2 parameters.

Reply:
No. I would like you to convert the only magic numbers that are made in this PR.